sphinx-mode.el

```diff
@@ -122,10 +123,49 @@ If BUFFER is not given use the `current-buffer'."
     (find-file (plist-get target :file))
     (goto-char (plist-get target :point))))
 
+(defun sphinx-compile-find-makefile-dir () ; adapted from rst-compile-find-conf
```

Comment:
I think this entire function can be replaced by the built-in 

`(locate-dominating-file (buffer-file-name) "Makefile")`.